src/components/RouteOption/DockedEbikeRouteOptionDetails.jsx

```diff
@@ -33,16 +33,28 @@ function DockedEbikeRouteOptionDetails({ routeOption }) {
     walkingTimeFromStartingPoint,
     cyclingTimeStationToStation,
     walkingTimeToDestination,
+    usualAvailabilityAtBikePickupStation,
+    usualAvailabilityAtBikeDropOffStation,
   } = routeOption.details;
 
+  const pickupStationCapacity =
+    (usualAvailabilityAtBikePickupStation?.standardBikes || 0) +
+    (usualAvailabilityAtBikePickupStation?.electricBikes || 0) +
+    (usualAvailabilityAtBikePickupStation?.emptyDocks || 0);
```

Comment:
Wow. I didn't know this was valid JavaScript - apparently, they added this in ES11

Very nice. Learning new things 😄